src/main/java/seedu/momentum/model/project/Description.java

```diff
@@ -0,0 +1,55 @@
+package seedu.momentum.model.project;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.momentum.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Project's Description in the project book.
+ * Guarantees: immutable; is valid
+ */
+public class Description {
+
+    public static final String MESSAGE_CONSTRAINTS = "Description can take any value, and it should not be blank";
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
```

Comment:
Would it be better for descriptions to be optional?

Reply:
Not sure what the others think about this, I think that would also work.